packages/rehype-mathjax/browser.d.ts

```diff
@@ -0,0 +1,12 @@
+import {Plugin} from 'unified'
+
+// http://docs.mathjax.org/en/latest/options/input/tex.html#the-configuration-block
+type MathNotation = [string, string]
+interface BrowserOptions {
+  displayMath: MathNotation | MathNotation[]
+  inlineMath: MathNotation | MathNotation[]
+}
+
+declare const renderBrowser: Plugin<[Partial<BrowserOptions>?]>
```

Comment:
There are some different approach to options across the unified/remark community.
We are normalizing toward using `?` for optional parameters in most cases, and using `Partial` only for external types that need to be made optional.
Since this (currently) an internal interface, `?` would be preferred.

Reply:
@uetchy friendly ping

Reply:
@ChristianMurphy Fixed now! https://github.com/remarkjs/remark-math/pull/48/commits/aef142dc8c9d275ec7296212b3653dd108de1986